app/components/CoinSwiper.tsx

```diff
@@ -37,7 +35,7 @@ export const CoinSwiper = ({ handleClick, combinedChartCoins }) => {
         <div>
           <Slider {...settings}>
             {top10Coins?.map((coin) => (
-              <div key={coin.id}>
+              <div key={coin?.id}>
```

Comment:
https://www.loom.com/share/73946853086648818aeed2719ac079cb?from_recorder=1&focus_title=1

Reply:
lol I probably did a replace all. question marks solve a lot of my problems but I can't imagine any reason I would see putting it in with the key, as the solution.